File: utils/linked_list.cpp
#include "linked_list.hpp" // header in local directory
#include <stdio.h>


/* Print all the elements in the linked list */
void linked_list::print_list(linked_list::ListNode *head) {
    linked_list::ListNode *current_node = head;
   	while ( current_node != NULL) {
        printf("%d ", current_node->val);
        current_node = current_node->next;
    }
}

File: utils/linked_list.py
class ListNode:
    def __init__(self, val=0, next=None):
        self.val = val
        self.next = next

def print_list(l:ListNode):
    res=[]
    while l!=None:
        res.append(str(l.val))
        l=l.next
    res=" ".join(res)
    print(res)
    return  res